src/components/SelectionListWithModal/index.tsx

```diff
@@ -23,13 +23,14 @@ function SelectionListWithModal<TItem extends ListItem>(
     const [isModalVisible, setIsModalVisible] = useState(false);
     const [longPressedItem, setLongPressedItem] = useState<TItem | null>(null);
     const {translate} = useLocalize();
-    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    // We need to use isSmallScreenWidth instead of shouldUseNarrowLayout https://github.com/Expensify/App/issues/48675
```

Comment:
Can you explain a bit more here WHY we need to use `isSmallScreenWidth`? I like the link to the issue, but it's better if we can make the comment clear enough so nobody has to go and open the issue